### PR TITLE
ColorfulScrollBar Successfully added

### PR DESCRIPTION
**Pull request by ROHAN842**
image

Fixes issue #64 

I have succesfully added colorful Scrollbar

Kindly merge my PR

### CODE REVIEW

**Comment by ROHAN842** [comment]
Hey @hkaur008 please merge my PR

**Comment by hkaur008** [comment]
This is not looking nyc wrt UI

File: src/components/texteditor.css [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
-margin-top: 5px;margin-bottom: 5px;
-padding:20px;
-
-overflow-y: auto;
+::-webkit-scrollbar {
+    width: 20px;
+}
 
+::-webkit-scrollbar-track {
+    background: linear-gradient(0deg, rgba(231, 248, 47, 1) 0%, rgba(39, 246, 190, 1) 50%, rgba(7, 0, 211, 1) 100%);
```
Comment (shreyshreyansh) [inline src/components/texteditor.css:20]: Use:
background : rgb(240, 166, 203)
That would look nice

**Comment by ROHAN842** [comment]
OK

**Comment by ROHAN842** [comment]
image

Hey @shreyshreyansh I have changed the color as you have given  background : rgb(240, 166, 203)

Now can you please merge my PR

**Comment by ROHAN842** [comment]
Hey @shreyshreyansh please merge my PR

**Comment by hkaur008** [comment]
hey I asked you to remove scroll bar not contact form . I think you did opposite

**Comment by ROHAN842** [comment]
Hey @hkaur008 can you merge my PR

**Pull request merged**